app/src/main/java/io/islnd/android/islnd/app/activities/SplashScreenActivity.java

```diff
@@ -0,0 +1,58 @@
+package io.islnd.android.islnd.app.activities;
+
+import android.accounts.Account;
+import android.accounts.AccountManager;
+import android.content.ContentResolver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Handler;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+
+
+import io.islnd.android.islnd.app.R;
+import io.islnd.android.islnd.app.database.IslndContract;
+import io.islnd.android.islnd.app.util.Util;
+
+public class SplashScreenActivity extends AppCompatActivity {
+
+    private static final String TAG = SplashScreenActivity.class.getSimpleName();
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_splash_screen);
+
+        // Create sync account and force sync
+        Account syncAccount = createAndRegisterSyncAccount(this);
+        ContentResolver resolver = getContentResolver();
+        resolver.setSyncAutomatically(
+                syncAccount,
+                getString(R.string.content_authority),
+                true);
+        Log.v(TAG, "requesting sync...");
+        resolver.requestSync(syncAccount, IslndContract.CONTENT_AUTHORITY, new Bundle());
+
+        Util.applyAppTheme(this);
+
+        // Visual pause...
+        Handler handler = new Handler();
+        Runnable runnable = () -> {
+            finish();
+            startActivity(new Intent(SplashScreenActivity.this, NavBaseActivity.class));
+        };
+
+        handler.postDelayed(runnable, 1500);
```

Comment:
I'm pausing the activity for 1.5 seconds until I open the next activity. This allows the user to actually see the splash. I'm not sure if this is bad form or not..